yang/frr-isisd.yang

```diff
@@ -881,6 +881,37 @@ module frr-isisd {
           description
             "This leaf describes the state of the interface.";
         }
+
+        container adjacency-sids {
```

Comment:
There's already an IETF draft https://datatracker.ietf.org/doc/html/draft-ietf-isis-sr-yang-11 that proposes another structure and I think it's better to use standard structure whenever possible.
```
       list adjacency-sid {
         key value;
         config false;
         leaf af {
           type iana-rt-types:address-family;
           description
             "Address-family associated with the
              segment ID";
         }
         leaf value {
           type uint32;
           description
             "Value of the Adj-SID.";
         }
         leaf weight {
           type uint8;
           description
             "Weight associated with
             the adjacency SID.";
         }
         leaf protection-requested {
           type boolean;
           description
             "Describe if the adjacency SID
              must be protected.";
         }
         description
           "List of adjacency Segment IDs.";
       }
```

Reply:
Strongly agree with @idryzhov.

Reply:
@idryzhov 
I'm very sorry for the delay in replying ...

I would like to make sure that we are on the same page about the data structure of yang, so I would like to ask your opinion if the following data structure is correct.
```
grouping adjacency-state {
  container adjacencies {
    list adjacency {
      container adjacency-sids {
        list adjacency-sid {
          leaf af{type iana-rt-types:address-family;}
          leaf value{type uint32;}
          leaf weight{type uint8;}
          leaf protection-requested{type boolean;}
        }
      }
    }
  }
}
```

[Post Script]
I made a commit 2723f30, which you may not have intended, so please ignore it.

Reply:
Yes. Define a grouping first and then use it in the container.

Reply:
The grouping adjacency-state was already implemented, so I defined the rest(container adjacency-sids and lan-adjacency-sids) at commit adb3a79 and 38ee5ec.
Would you please check them?